data/human/human missions.txt

```diff
@@ -6301,3 +6301,71 @@ mission "Blind Man from Martini"
 				clear "Blind Man from Martini: label looking"
 			`	With that, he grabs his cane and his luggage, and follows a hospital attendant toward his daughter's room, leaving you in the reception.`
 
+
+
+mission "FW Refugees to Humanika"
+	name "Transport Refugee Family"
+	description "Transport a family of <bunks> refugees and <cargo> to <destination>. Payment is <payment>."
+	minor
+	source
+		attributes "dirt belt" rim south
+		not government Quarg
+		not near Tarazed 1
+	destination Humanika
+	to offer
+		has "event: war begins"
+		not "chosen sides"
+		random < 15
+	to fail
+		has "event: joined the free worlds"
+	passengers 3
+	cargo "household goods" 2
+	on offer
+		conversation
+			`A young family of three is walking around the spaceport talking to merchant captains. From the looks of it, they're not having much luck.`
+			choice
+				`	(Approach them.)`
+				`	(Ignore them.)`
+					defer
+			`	The parents introduce themselves as Penny Little and Rosa Perkins. Their young child is named Eduardo. "Any chance you're willing to take us and <tons> of our belongings to the Quarg planet <destination>? We're concerned that the war between the Republic and the Free Worlds is going to get dangerous, and we want to raise Eduardo somewhere safe. We can pay <payment>."`
+			label "main choice"
+			choice
+				`	"Why were all those other captains refusing to take you?"`
+					to display
+						not "FW Refugees to Humanika: other captains"
+					goto "other captains"
+				`	"What do you mean, 'Quarg planet'?"`
```

Comment:
```suggestion
				`	"What do you mean, 'Quarg planet?'"`
```

Reply:
See https://github.com/endless-sky/endless-sky/pull/9159#discussion_r1299249287 above, I'm pretty sure this is correct.

Reply:
This is actually an American vs British English thing. The game is written in American English; as such, the `?` should be inside both quotation marks.

Reply:
That's for commas and periods, not question marks, I think. I think question marks are the same in both styles. E.g., https://www.thepunctuationguide.com/british-versus-american-style.html says:

> The above examples also show that the American style places commas and periods inside the quotation marks, even if they are not in the original material. British style (more sensibly) places unquoted periods and commas outside the quotation marks. For all other punctuation, the British and American styles are in agreement: unless the punctuation is part of the quoted material, it goes outside the quotation marks.

P.S. I'm American btw, though I read enough English from other places that I don't always write in American style without consciously thinking about it.